### PR TITLE
Dark mode support for portal UI

### PR DESCRIPTION
**Pull request by ashleysmithTTD**
What changed:

- uncommented dark mode code
- added several new themes and changes to css files to make the color alternate between normal and dark mode

### CODE REVIEW

File: .storybook/main.ts
Comment (alex-yau-ttd) [inline .storybook/main.ts:1]: Should we just revert this change?

**Review by alex-yau-ttd — APPROVED** [review]

File: src/web/utils/errorHandler.scss [hunk 1]
```diff
@@ -1,5 +1,31 @@
-.error-content {
-  align-self: flex-start;
-  text-align: center;
-  padding: 4.5rem 6.5rem;
+.app {
+  .error-content {
```
Comment (alex-yau-ttd) [inline src/web/utils/errorHandler.scss:2]: We could probably extract this part out so we don't need to define it twice.
Something like:
```
.error-content {
  // common stuff
}

.app .error-content {
  // light mode specific stuff
}

.app.darkmode .error-content {
 // dark mode specifif stuff
}
```

File: .storybook/main.ts
Comment (ashleysmithTTD) [reply .storybook/main.ts:1]: I'm not sure how to override my editor - everytime i try to save the file, it auto makes this format changes
